docs/data-sources/cassandra_cluster_v2_instance.md

```diff
@@ -143,14 +143,20 @@ The version of Shotover Proxy running on the Cluster.<br>
 ### Read-only attributes
 *___version___*<br>
 <ins>Type</ins>: string, read-only<br>
-<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>The version of Shotover Proxy running on the Cluster.Available versions: <ul> <li>`0.1.10`</li> </ul><br><br>
```

Comment:
I think this shouldn't be in this PR

Reply:
Yeah, the docs are auto-generated, I think it should be fine to include in this PR. Team Cassandra might forget the terraform-provider deployment.